contracts/cw20-base/Cargo.toml

```diff
@@ -28,6 +28,7 @@ schemars = "0.8.1"
 semver = "1"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 thiserror = { version = "1.0.23" }
+serde_json = "1.0.85"
```

Comment:
We specifically avoid using `serde_json` in our code and use [`serde_json_wasm`](https://github.com/CosmWasm/serde-json-wasm) instead. This is for efficiency and determinism as far as I recall.

Reply:
Thanks, I will update